### PR TITLE
load hotspot details from server and reder cats

### PR DESCRIPTION
**Pull request by mveres**
https://user-images.githubusercontent.com/7051248/156364651-a729dfb7-71df-4a3e-891b-22d10e9816db.mov

### CODE REVIEW

File: components/CatCard.tsx [hunk 1]
```diff
@@ -127,56 +130,73 @@ export const CatCard = (props: CatCardProps) => {
...
             </View>
           </View>
         )}
-        {!isEmpty(props.cat.media) && (
+        {!isEmpty(cat.media) && (
           <FlatList
             style={{ marginTop: 20 }}
             horizontal
-            data={props.cat.media}
+            data={[] /*cat.media*/}
```
Comment (debreb) [inline components/CatCard.tsx:199]: [] is added with a purpose?

File: components/CatCard.tsx [hunk 2]
```diff
@@ -127,56 +130,73 @@ export const CatCard = (props: CatCardProps) => {
...
+            <Text style={styles.title}>Sex</Text>
           </View>
           <View style={styles.circleView}>
-            <Text style={styles.genderText}>{props.cat.sex}</Text>
+            <Text style={styles.genderText}>{cat.sex}</Text>
           </View>
         </View>
-        {props.cat.notes && (
-          <Text style={styles.notes}>Observatii: {props.cat.notes}</Text>
+        {!!cat.notes && (
```
Comment (debreb) [inline components/CatCard.tsx:140]: If `notes` is a required field, it's necessary to the `!!` operator?

**Review by debreb — APPROVED** [review]

File: components/CatCard.tsx (hunk 2 above)
Comment (mveres) [reply components/CatCard.tsx:140]: Notes might be empty, and then it leaves a lot of white space in the screen due to the padding and margin.
So I'd rather we don't render the text field at all when `notes` is empty for visual purposes.

File: components/CatCard.tsx (hunk 1 above)
Comment (mveres) [reply components/CatCard.tsx:199]: Yes, for now `cat.media` is not an array, it's an object with some links keys inside and the FlatList won't accept it. Until we have a better data model, I replaced with an empty array.

**Pull request merged**